linux_os/guide/system/logging/rsyslog_sending_messages/rsyslog_remote_tls/bash/shared.sh

```diff
@@ -0,0 +1,23 @@
+# platform = multi_platform_fedora,multi_platform_ol,multi_platform_rhel,multi_platform_sle
+# reboot = false
+# strategy = configure
+# complexity = low
+# disruption = low
+
+{{{ bash_instantiate_variables("rsyslog_remote_loghost_address") }}}
+
+# Get omfwd configuration directive
+OMFWD_CONFIG_OUTPUT=`grep -Pzo '(?s)action\s*\(\s*type\s*=\s*"omfwd".*\)' /etc/rsyslog.conf /etc/rsyslog.d/*.conf`
+OMFWD_CONFIG=`echo "$OMFWD_CONFIG_OUTPUT"| awk 'BEGIN {FS=":"; RS=")\n"}; {print $2}'`
+OMFWD_CONFIG_FILE=`echo OMFWD_CONFIG_OUTPUT| awk 'BEGIN {FS=":"; RS=")\n"}; {print $1}'`
+if [! -z "$OMFWD_CONFIG" ]; then
+    OMFWD_TLS_STREAM=`grep  ' StreamDriver="gtls" ' "$OMFWD_CONFIG"`
+    if [! -z "!OMFWD_TLS_STREAM" ]; then
+        exit 0
+    else
+        # insert TLS stream param
+        sed -i 's/action\s*(\s*type\s*=\s*"omfwd"/action(type="omfwd"\ StreamDriver="gtls"\ /' $OMFWD_CONFIG_FILE
+    fi
+else
+    echo "action(type=\"omfwd\" protocol=\"tcp\" Target=\"{{ rsyslog_remote_loghost_address }}\" port=\"6514\" StreamDriver=\"gtls\" StreamDriverMode=\"1\" StreamDriverAuthMode=\"x509/name\" streamdriver.CheckExtendedKeyPurpose=\"on\")"  >> /etc/rsyslog.conf
```

Comment:
Instead of Jinja variable here you need to use the bash variable that you created by Jinja macro on line 7.

```suggestion
    echo "action(type=\"omfwd\" protocol=\"tcp\" Target=\"$rsyslog_remote_loghost_address\" port=\"6514\" StreamDriver=\"gtls\" StreamDriverMode=\"1\" StreamDriverAuthMode=\"x509/name\" streamdriver.CheckExtendedKeyPurpose=\"on\")"  >> /etc/rsyslog.conf
```

Reply:
Thx should be ok in [6453b4d](https://github.com/ComplianceAsCode/content/pull/9484/commits/6453b4d70b9f274bfef9a2ed00df65c4e9a68c51)